src/zarr/testing/buffer.py

```diff
@@ -51,6 +53,15 @@ def create(
             ret.fill(fill_value)
         return ret
 
+    @classmethod
+    def empty(
+        cls,
+        shape: ChunkCoords,
+        dtype: npt.DTypeLike,
+        order: Literal["C", "F"] = "C",
+    ) -> Self:
+        return super(cpu.NDBuffer, cls).empty(shape=shape, dtype=dtype, order=order)
```

Comment:
This is to test the base `core.NDBuffer` implementation. This class inherits from `cpu.NDBuffer` so we use `super(cpu.NDBuffer, cls)` to get its parents (the base class).